packages/enzyme/src/selectors.js

```diff
@@ -160,6 +160,15 @@ function matchPseudoSelector(node, token, root) {
     const { rendered } = findParentNode(root, node);
     return rendered[rendered.length - 1] === node;
   }
+  if (name === 'focus') {
+    if (typeof document === 'undefined') {
+      throw new Error('Enzyme::Selector does not support ":focus" pseudo-element without global "document".');
+    } else {
```

Comment:
Since this is a throw, there's no need for the "else"
```suggestion
    }
```